kde-builder: add page (#13449)
tldr-pages/tldr: pages/linux/kde-builder.md

# kde-builder

> Easily build KDE components from its source repositories.

> > Drop-in replacement for kdesrc-build.

Reviewer: ```suggestion
> Drop-in replacement for `kdesrc-build`.
```

> More information: <https://kde-builder.kde.org/en/cmdline/cmdline-usage.html>.

- Initialize `kde-builder`:

`kde-builder --initial-setup`

- Compile a KDE component and its dependencies from the source:

`kde-builder {{component_name}}`

> - Compile a component without updating its local code and without compiling its dependencies:

Reviewer 2: ```suggestion
- Compile a component without updating its local code and without compiling its [D]ependencies:
```

`kde-builder --no-src --no-include-dependencies {{component_name}}`

> - Refresh the build directories before compiling:

Reviewer 2: ```suggestion
- [r]efresh the build directories before compiling:
```
